src/fides/data/sample_project/sample_connections/sample_connections.yml

```diff
@@ -0,0 +1,48 @@
+connection:
```

Comment:
FWIW, this file is a very useful side-effect of this refactor. It makes it possible to trivially add a connector to the test environment for testing and inject all the necessary secrets for it as ENV vars, and do so in a pretty declarative fashion.

The format here is designed to mimic the `PATCH /api/v1/connections/` endpoint directly, but with the notable addition of the `dataset` field which isn't supported by that API.

Reply:
This is the best part! We didn't have a good way to set up a consent connector automatically before, could add this if interested https://github.com/ethyca/fides/pull/3104/files